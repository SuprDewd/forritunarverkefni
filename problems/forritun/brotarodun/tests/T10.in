10
27/735
823/896
458/719
485/18
159/893
936/769
431/491
694/656
162/664
981/897
