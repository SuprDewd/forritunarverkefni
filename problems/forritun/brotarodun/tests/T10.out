27/735
159/893
162/664
458/719
431/491
823/896
694/656
981/897
936/769
485/18
